Example/ParalayoutSnapshotTests/ViewDistributionSnapshotTests.swift

```diff
@@ -121,6 +121,59 @@ final class ViewDistributionSnapshotTests: SnapshotTestCase {
         verifySnapshot(margin: 40, inRect: CGRect(x: 20, y: 10, width: 300, height: 50))
     }
 
+    func testDistribution() {
```

Comment:
We should really have more thorough snapshot tests for view distribution, but I added what I felt was necessary to verify the changes in this PR here. We should follow up separately with building out the test suite.

Reply:
That would be helpful. I think this solution makes sense, but it's hard to visualize any edge cases.